src/wp-content/themes/twentytwentyfour/patterns/features-with-images.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Title: Features with Images
+ * Slug: twentytwentyfour/features-with-images
+ * Categories: text
+ * Viewport width: 1400
+ */
+?>
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"var:preset|spacing|50","bottom":"var:preset|spacing|50","left":"var:preset|spacing|50","right":"var:preset|spacing|50"},"margin":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignfull" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50)"><!-- wp:group {"align":"wide","style":{"spacing":{"blockGap":"0"}},"layout":{"type":"constrained"}} -->
+<div class="wp-block-group alignwide"><!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"flex","orientation":"vertical","justifyContent":"center"}} -->
+<div class="wp-block-group">
+
+<!-- wp:heading {"textAlign":"center","className":"is-style-asterisk"} -->
+<h2 class="wp-block-heading has-text-align-center is-style-asterisk"><?php echo esc_html_x( 'An array of resources', 'Sample content for heading of the section.', 'twentytwentyfour' ); ?></h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph {"align":"center","style":{"layout":{"selfStretch":"fit","flexSize":null}}} -->
+<p class="has-text-align-center"><?php echo esc_html_x( 'Our comprehensive suite of professional services caters to a diverse clientele, ranging from homeowners to commercial developers.', 'Sample content for the subheading for this pattern.', 'twentytwentyfour' ); ?></p>
```

Comment:
I think that the context parameter for this translation will need to be more specific. "Sample content for the subheading for this pattern" will not make sense to a translator out of context. They won't know which pattern. There's likely a handful of others that need updating to reference which pattern the string belongs to.

Reply:
I opened an [issue](https://github.com/WordPress/twentytwentyfour/issues/505) on the repo for this, since there's probably a few strings that need context that currently don't have it and it's probably going to be a big audit